### ISSUE TITLE
Screen reader is not announcing Complete information displayed while navigating on the chart under 'Browse market shares'.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#19384]
#### Expected behaviour
Along with tooltip NVDA should announce x-axis and y-axis information, NVDA should announce as 'Total percentage of market shares in x-axis and Browser in y-axis, 'Browser market shares. January 2022. Highcharts interactive chart. region Chrome, 63.06. Browsers. graphic    Interactive chart graphic Created with Highcharts 11.0.1 '

#### Actual behaviour
When navigating on the chart NVDA announces as 'Browser market shares. January 2022. Highcharts interactive chart. region Chrome, 63.06. Browsers. graphic    Interactive chart graphic Created with Highcharts 11.0.1' and it's not giving the x-axis and y-axis information to the users.

#### Live demo with steps to reproduce

1.Open URL: [Column with drilldown demo | Highcharts.com](https://www.highcharts.com/demo/column-drilldown)  in Edge and 
   login with valid credentials
2. Navigate to the 'Browse market shares' chart by using tab key.
3. Observe the screen reader narrating all the information for the market share chart or not.

#### Product version

Edge Version 114.0.1823.79 (Official build) (64-bit)
Window 11 22H2 (OS Build 22621.1848)

#### Affected browser(s)
Edge Version 114.0.1823.79 (Official build) (64-bit)

Attachments:

https://github.com/highcharts/highcharts/assets/93735775/51ac9104-0f8a-4417-b0c3-86e76a0abc5f

**Comment by msftedad** [comment]
Hi Team, Any update on this issue? Please provide the ETA to fix this issue.

**Comment by oysteinmoseng** [comment]
Hi @msftedad, I don't think this is a bug. If you navigate using the Tab key, you will skip all the non-interactive elements, including the description before the chart where the X and Y axes are described.

If you would like to change the accessible name of the whole interactive chart you can do so through [the API options](https://api.highcharts.com/highcharts/lang.accessibility.chartContainerLabel), but I would only recommend this if you have a very short description. 

Closing for now, but feel free to comment to reopen.
